### PR TITLE
Azure virtual network

### PR DESCRIPTION
**Pull request by JunqiZhang0**
Added virtual network creation and destruction

### CODE REVIEW

File: docs/source/azure.rst [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+| role             | true       | string        | N/A               |                 |
++------------------+------------+---------------+-------------------+-----------------+
+| vm_name          | true       | string        | name              |It can't include |
+|                  |            |               |                   | _ and other     |
+|                  |            |               |                   |special char     |
++------------------+------------+---------------+-------------------+-----------------+
+| image            | true       | string        | image             |                 |
++------------------+------------+---------------+-------------------+-----------------+
+
+EC2 Inventory Generation
```
Comment (abraverm) [inline docs/source/azure.rst]: Remove AWS related documentations

**Comment by abraverm** [comment]
squash please

**Comment by abraverm** [comment]
@JunqiZhang0 - conflicts

**Comment by samvarankashyap** [comment]
@JunqiZhang0 Please resolve conflicts before you merge.

**Comment by JunqiZhang0** [comment]
[merge]

**Comment by JunqiZhang0** [comment]
[merge]

**Pull request merged**